_episodes/04-use-license.md

```diff
@@ -5,7 +5,185 @@ exercises: 0
 questions:
 - "Key question"
 objectives:
-- "First objective."
+- Tell what is a copyright and what a licence does
+- tell why is important that a product/code has a licence
+- tell what is the importance of third-party dependencies on your product/code
+- Choose a licence for your code
+- Add a LICENCE file to a repository
 keypoints:
 - "First key point."
 ---
+
+## Tell what is public domain, a copyright and what a licence does
+
+### What is a copyright
+
+* Copyright is a legal right over the creative expression of an idea or an intellectual work. It grants the creator of an original work exclusive rights to determine and decide whether, and under what conditions, this original work may be used by others. 
+* Copyright is automatically attached to every novel expression of an idea, including a poem or a blog entry, for a certain amount of years, typically from 50 to 100 years after the idea creator dies (depending on the jurisdiction). That is, the law assumes that as the author of your code, you have a say in what others can do with it.
+* In particular, the code that you write is copyrighted by default in the most restrictive way. This implies that nobody, except you, can use, modify (even to fix bugs!), create derivative works, distibute it.
+* If you infringe on someone's copyright you may  be in the situation of having to pay reasonable damages.
+* "Fair rules" - parody, criticism, quotation - are an exception to these rights. For example, you can publish a study comparing the performance or the accuracy of several different programs, without infringing the copyright of the programs.
+
+> ### NOTE: Copyright vs Patent vs Trademarks
+>
+> **Copyright** protects the creative expression of ideas, **patents** protect novel technological inventions, **trademarks** protect ownership.
+{: .callout}
+
+> ### Challenge
+>
+> For each one of the three scenarios below, answer yes or no to the question.
+> 
+> 1. **Scenario:** you receive a sticker and your friends want a copy.
+> 
+>    **Question:** can you make a copy of the sticker and sell the copy to your friends?
+> 2. **Scenario:** you receive a script from your supervisor asking you to modify it to analyse a different type of input data.
```

Comment:
Did the supervisor write the script? This was unclear to me from the wording. Perhaps 

> Your supervisor sends you a script that they wrote, asking you to modify [...]

Or

> Abdul's supervisor sends him a script that she wrote, asking him to [...]

Reply:
Usually, you and your supervisor would be working for the same entity on a work-for-hire basis, so is this really a copyright issue? That may actually be interesting to discuss here, so it's not a bad example at all, but maybe it should be mentioned somewhere?